scripts/vars.sh

```diff
@@ -65,6 +65,7 @@ HOST_WEEK_EPOCH_DURATION="60s"
 UNBONDING_TIME="120s"
 MAX_DEPOSIT_PERIOD="30s"
 VOTING_PERIOD="30s"
+HARSHEN_SLASHING=false
```

Comment:
can you add a slightly longer comment describing what this does?

Reply:
I can - although, do you think we should just enable these by default? I think you have to manually kill containers for any of these configurations to matter